### PR TITLE
International Address Form pattern edits

### PR DESCRIPTION
**Pull request by mikearildbrown**

### CODE REVIEW

File: site/docs/patterns/international-address-input.mdx [hunk 1]
```diff
@@ -145,6 +144,6 @@ You can add a switch to toggle localization after the country selection, enablin
   />
 </Diagrams>
 
-## Expanding the pattern
+## Feedback
```
Comment (joshwooding) [inline site/docs/patterns/international-address-input.mdx:147]: Is this going to be standard across all the pages? If so we should just create a component to template it.

File: site/docs/patterns/international-address-input.mdx [hunk 2]
```diff
@@ -1,45 +1,76 @@
 ---
-title: International address input
+title: International address form
 layout: DetailPattern
 aliases:
   - /salt/patterns/international-address-input
```
Comment (joshwooding) [inline site/docs/patterns/international-address-input.mdx:5]:
```suggestion
  - /salt/patterns/international-address-form
```

File: site/docs/patterns/international-address-form.mdx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  />
+
+## Dynamic format
+
+The dynamic format uses form field labels specific to a country or region. The form only displays fields relevant to a specific country in a recognizable order and with familiar labels. This will reduce barriers to input and minimize user errors.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/international-address-input/iai-dynamic-format-toggle-off.png",
```
Comment (joshwooding) [inline site/docs/patterns/international-address-form.mdx:70]: I think this image needs to be updated.

**Review by joshwooding — CHANGES_REQUESTED** [review]
See comment

**Review by shey-v — APPROVED** [review]

**Review by shey-v — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**